sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -707,24 +742,25 @@ To learn more about other logging mechanisms see [here][logging].
 Samples showing how to use this client library are available in this GitHub repository.
 Samples are provided for each main functional area, and for each area, samples are provided for analyzing a single document, and a collection of documents in both sync and async mode.
 
-- [Detect Language][detect_language_sample]
-- [Analyze Sentiment][analyze_sentiment_sample]
-- [Extract Key Phrases][extract_key_phrases_sample]
-- [Recognize Named Entities][recognize_entities_sample]
-- [Recognize PII Entities][recognize_pii_entities_sample]
-- [Recognize Linked Entities][recognize_linked_entities_sample]
-- [Recognize Healthcare Entities][analyze_healthcare_sample]
-- [Perform Extractive Text Summarization][extract_summary_sample]
-- [Custom Named Entity Recognition][recognize_custom_entities_sample]
-- [Custom Single Category Classification][single_category_classify_sample]
-- [Custom Multi Category Classification][multi_category_classify_sample]
+* [Detect Language][detect_language_sample]
+* [Analyze Sentiment][analyze_sentiment_sample]
+* [Extract Key Phrases][extract_key_phrases_sample]
+* [Recognize Named Entities][recognize_entities_sample]
+* [Recognize PII Entities][recognize_pii_entities_sample]
+* [Recognize Linked Entities][recognize_linked_entities_sample]
+* [Recognize Healthcare Entities][analyze_healthcare_sample]
+* [Custom Named Entity Recognition][recognize_custom_entities_sample]
+* [Custom Single Category Classification][single_category_classify_sample]
```

Comment:
Custom Single **Label** Classification and Custom Multi **Label** Classification